lib/jobs/ansible-job.js

```diff
@@ -107,11 +97,27 @@ function ansibleJobFactory(
                 logger.warning("Result file not found. " + err.message);
                 self._done();
             } else {
-                self._done(err);
+                loadResultFile(self.outFileName,self.context)
+                .then(function() {
+                    self._done(err);
+                });
```

Comment:
Might be good to add a `.catch` on here as well, in case your assert statement fails in `loadResultFile`? Otherwise there's a possibility for silent failures.